<launch>

  <node pkg="line_detection" name="histogram_calculator_lanedetection" type="histogram-calculator.py" output="screen">
    
    <param name="use_compressed_format" type="bool" value="true" />
    <param name="subscriber_image_topic" type="str" value="/camera/image_raw" />
    <param name="buffer_size" type="int" value="52428800" />
    <param name="histogram_dimension" type="int" value="50" />

  </node>
  
  <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />

</launch>